src/main/java/pers/gwyog/gtneioreplugin/util/GT5OreLayerHelper.java

```diff
@@ -12,11 +14,28 @@
 
 public class GT5OreLayerHelper {
 
+    class oreDimensionWrapper {
+        ArrayList<OreLayerWrapper> internalDimOreList = new ArrayList<>();
+        HashMap<OreLayerWrapper, Double> oreVeinToProbabilityInDimension = new HashMap<OreLayerWrapper, Double>();
```

Comment:
1.
 ```java
List<OreLayerWrapper> internalDimOreList = new ArrayList<>();
Map<OreLayerWrapper, Double> oreVeinToProbabilityInDimension = new HashMap<>();
```
2. this, too, should be either public or private
3. and probably final too